_sun.scss

```diff
@@ -8,3 +44,38 @@
 	box-shadow: inset $sun-shadow-size -#{$sun-shadow-size} 0px 0px $sun-shadow;
 	transform: translate(-50%, -50%);
 }
+
+.sun.shining {
```

Comment:
We don't need new class - we can use descendant selector: `.solar-system--3d .sun` for the same result. What do you think about this solution?